src/explore-education-statistics-admin/src/modules/find-statistics/components/KeyStatistics.tsx

```diff
@@ -0,0 +1,191 @@
+                                    contentBlock => {
+                                      if (contentBlock.id === stat.id) {
+                                        return {
+                                          ...stat,
+                                          summary: {
+                                            dataKeys:
+                                              (stat.summary &&
+                                                stat.summary.dataKeys) ||
+                                              [],
+                                            ...values,
+                                          },
+                                        };
+                                      }
+                                      return contentBlock;
+                                    },
+                                  )
+                                : [],
+                            },
+                          });
+                        }),
+                    );
+                  }}
+                />
+              ) : null;
+            }
+            return null;
+          })}
+      </div>
+    </>
+  );
+};
+
+const AddKeyStatistics = ({ release, setRelease }: KeyStatisticsProps) => {
+  const { updateAvailableDataBlocks } = useContext(EditingContext);
+  const [isFormOpen, setIsFormOpen] = useState<boolean>(false);
+
+  const another =
+    release.keyStatisticsSection.content &&
+    release.keyStatisticsSection.content.length > 0 &&
+    ' another ';
+  return (
+    <>
+      {isFormOpen && (
+        <KeyIndicatorSelectForm
+          onSelect={async datablockId => {
+            if (
+              release.keyStatisticsSection &&
+              release.keyStatisticsSection.id
+            ) {
+              await releaseContentService
+                .attachContentSectionBlock(
+                  release.id,
+                  release.keyStatisticsSection &&
+                    release.keyStatisticsSection.id,
+                  {
+                    contentBlockId: datablockId,
+                    order:
+                      (release.keyStatisticsSection.content &&
+                        release.keyStatisticsSection.content.length) ||
+                      0,
+                  },
+                )
+                .then(v => {
+                  if (updateAvailableDataBlocks) {
+                    updateAvailableDataBlocks();
+                  }
+                  return v;
+                });
+
+              const keyStatisticsSection = await releaseContentService.getContentSection(
+                release.id,
+                release.keyStatisticsSection.id,
+              );
+              if (keyStatisticsSection) {
+                setRelease({
+                  ...release,
+                  keyStatisticsSection,
+                });
+                setIsFormOpen(false);
+              }
+            }
+          }}
+          onCancel={() => setIsFormOpen(false)}
```

Comment:
Do we need to reset any values when the form closes so they're not remembered the next time it's opened?

Reply:
Not required. there isn't actually a form, it's just one input that will reset when it renders